src/Curl/Curl.php

```diff
@@ -96,6 +98,8 @@ public function __construct($base_url = null)
 
         $this->curl = curl_init();
         $this->id = 1;
+        $this->responseParsing = true;
```

Comment:
I don't understand why you wouldn't just have the default to `true`.  By having them set in here, you may walk on an inheriting class.